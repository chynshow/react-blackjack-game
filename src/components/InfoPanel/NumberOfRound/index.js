import React from 'react';

const NumberOfRound = ({ gameRound, className }) => {
  return <div className={className}>Round: {gameRound} / 5</div>;
};

export default React.memo(NumberOfRound);
