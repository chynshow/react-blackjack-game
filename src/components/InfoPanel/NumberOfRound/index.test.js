import React from 'react';
import NumberOfRound from './index';
import { shallow } from 'enzyme';

it('render NumberOfRound component', () => {
  const c = shallow(<NumberOfRound />);
  expect(c).toMatchSnapshot();
});
