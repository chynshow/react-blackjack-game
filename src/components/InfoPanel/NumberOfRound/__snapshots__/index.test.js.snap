// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render NumberOfRound component 1`] = `
<div>
  Round: 
   / 5
</div>
`;
